lib/package_provider/repository.rb

```diff
@@ -31,7 +31,7 @@ def self.temp_prefix=(tp)
     end
 
     def self.temp_prefix
-      @temp_prefix || 'pp_repo_'
+      @temp_prefix || "#{Process.pid}_instance_pid_pp_repo_"
```

Comment:
it's quite long dir name, isn't it? the part "instance_pid" is meaning less I think.
I agree with PID in the dir name, but prefer "package_repo_#{process.pid}" or what ever